### PR TITLE
chore(deps): update dependencies

### PR DESCRIPTION
**Pull request by anehx**
BREAKING CHANGE: Remove support for Ember v3.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,8 +13,10 @@
     "lint:hbs:fix": "ember-template-lint . --fix",
     "lint:js": "eslint . --cache",
     "lint:js:fix": "eslint . --fix",
-    "lint:prettier": "prettier --check \"**/*.{scss,graphql,json,md,yaml}\"",
-    "lint:prettier:fix": "prettier --write \"**/*.{scss,graphql,json,md,yaml}\"",
+    "lint:css": "stylelint \"**/*.scss\"",
+    "lint:css:fix": "concurrently \"npm:lint:css -- --fix\"",
+    "lint:prettier": "prettier -c \"**/*.{graphql,json,md,yaml}\"",
+    "lint:prettier:fix": "prettier -wl \"**/*.{graphql,json,md,yaml}\"",
```
[the remark this answers is not in the document]
Comment (anehx) [reply package.json:19]: Absolutely not the same thing. Prettier only handles formatting, stylelint is a proper linter which is included per default in new ember addons. Stylelint is configured to use prettier for formatting so there is no change to the previous behaviour

[the remark this answers is not in the document]
Comment (derrabauke) [reply package.json:19]: :man_facepalming: totally right... Was just wondering what the motivation was. So it's a new default I very much welcome. :)

**Review by derrabauke — APPROVED** [review]
:rock: solidè!

**Pull request merged**